### PR TITLE
InvalidBackgroundColor flickering

### PR DESCRIPTION
**Pull request by grimes2**
In entry editor saving an invalid argument in a field causes a InvalidArgumentException. Using Ctrl-s or File>Save database the field flickers red only for a short moment. This is fixed and the three possibilities to save database are now consistent.

- [x] Change in CHANGELOG.md described
- [x] Manually tested changed features in running JabRef

### CODE REVIEW

**Comment by grimes2** [comment]
Please label "ready-to-review". Thanks.

File: src/main/java/net/sf/jabref/gui/fieldeditors/FieldEditorFocusListener.java [hunk 1]
```diff
@@ -23,7 +23,7 @@ public void focusGained(FocusEvent event) {
     @Override
     public void focusLost(FocusEvent event) {
         if (event.getSource() instanceof FieldEditor) {
-            ((FieldEditor) event.getSource()).setValidBackgroundColor();
+            //Do nothing
```
Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/gui/fieldeditors/FieldEditorFocusListener.java]: I cannot comment if this is the correct way to implement the problem right now, but if it is please rephrase the condition to 
```java
if (!(event.getSource() instanceof FieldEditor)) {
...
}
```

**Comment by stefan-kolb** [comment]
@grimes2 Can you please address the comments?

**Comment by tobiasdiez** [comment]
LGTM :+1: (without actually having tested the code)

Could you please rebase so that we can merge it? Thanks

**Comment by grimes2** [comment]
Sorry I missed that. Please merge.

**Pull request merged**